apps-rendering/package-lock.json

```diff
@@ -5974,7 +5974,7 @@
         "find-up": {
           "version": "2.1.0",
           "resolved": "https://registry.npmjs.org/find-up/-/find-up-2.1.0.tgz",
-          "integrity": "sha1-RdG35QbHF93UgndaK3eSCjwMV6c=",
```

Comment:
We really need to look into why these keep switching between sha1 and sha512 - it's very weird @JamieB-gu @MarSavar